### PR TITLE
Fix config functions action

### PR DESCRIPTION
**Pull request by schaefi**
The action failed on the setup of the runtime because the upgrade of pip failed.

### CODE REVIEW

File: .github/workflows/ci-config-functions.yml [hunk 1]
```diff
@@ -33,17 +33,13 @@ jobs:
           path: |
             ~/.cache/pip
             ~/.cache/pypoetry
-          key: pip-${{ hashFiles('.virtualenv.dev-requirements.txt') }}-${{ hashFiles('.virtualenv.requirements.txt') }}
-          restore-keys: |
-            -pip-
```
Comment (dcermak) [inline .github/workflows/ci-config-functions.yml:38]: Why did you remove this? It ensures that the action restores the correct cache

Comment (schaefi) [reply .github/workflows/ci-config-functions.yml:38]: These files don't exist anymore since we moved to poetry.

**Review by Conan-Kudo — APPROVED** [review]
